### DISCUSSION TITLE
1337x not supported

### DISCUSSION THREAD

**Discussion by TechJock** [qbittorrent/search-plugins#228] [Q&A]
Hello all,

When attempting to install 1337x, I'm informed that it is not supported.

BTW - I see that this was also raised by @at-prime back in Aug 2021.

Anyone know anything about this and/or how to fix?

Cheers!

![image](https://user-images.githubusercontent.com/58238005/224747654-289b57c5-8a00-41ae-83d6-98a4240fde84.png)

**Comment by ngosang** [comment]
It's an unofficial plugin. You will have to contact the authors: [sa3dany](https://github.com/sa3dany), [Alyetama](https://github.com/Alyetama), [BurningMop](https://github.com/BurningMop)

**Comment by aristsakas-ui** [comment]
i just uploaded the script today in discussion. it's tested and working. You just need to install one more lib. Read the readme for how to do that. Please post feedback but i think it is good and working
